src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Square extends Default {
+    private double size;
+
+    public Square(String color,double size) {
```

Comment:
Consider adding a space after the comma in the constructor parameters for better readability: 'String color, double size'.